decidim-core/spec/system/homepage_spec.rb

```diff
@@ -287,6 +287,53 @@
         end
       end
 
+      context "when organization forces users to authenticate before access" do
+        let(:organization) do
+          create(
+            :organization,
+            official_url:,
+            force_users_to_authenticate_before_access_organization: true
+          )
+        end
+
+        context "when there are site activities and user is not authenticated" do
+          let(:participatory_space) { create(:participatory_process, organization:) }
+          let(:component) do
+            create(:component, :published, participatory_space:)
+          end
+          let(:commentable) { create(:dummy_resource, component:) }
+          let(:comment) { create(:comment, commentable:) }
+          let!(:action_log) do
+            create(:action_log, created_at: 1.day.ago, action: "create", visibility: "public-only", resource: comment, organization:, participatory_space:)
+          end
+
+          before do
+            visit current_path
+            find_by_id("main-dropdown-summary").hover
+          end
+
+          it "does not show last activity section on menu bar main dropdown" do
+            expect(page).to have_no_content(translated(comment.body))
+            expect(page).to have_no_link("New comment")
+            expect(page).to have_no_link("Last activity")
+          end
+        end
+
+        context "when there is a promoted participatory space and user is not authenticated" do
+          let!(:participatory_space) { create(:participatory_process, :promoted, title:, organization:) }
+          let(:title) { {en: "Promoted, promoted, promoted!!!"} }
```

Comment:
To fix the rubocop offense: 

```suggestion
          let(:title) { { en: "Promoted, promoted, promoted!!!" } }
```